lib/Backends/OpenCL/OpenCL.h

```diff
@@ -176,8 +176,21 @@ class OCLBackend final : public Backend {
   bool transformPostLowering(Function *F, CompilationMode mode) const override;
 
   bool isOpSupported(Kinded::Kind opKind, ElemKind elementTy) const override {
+    // Check quantization support.
     if (elementTy == ElemKind::Int8QTy) {
```

Comment:
have you tried profiling Resnet50 graph with Interpreter and then load_profile with OpenCL backend?
Would be a good thing to look at and also dump DAG and see what was actually quantized.